recipes/sycomore/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set owner = "lamyj" %}
+{% set name = "sycomore" %}
+{% set version = "0.6.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ owner }}/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 963eaf04ae216c61fc212bda01a8dd2be188e2288b2a18c8c9a3f6209944590f
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<=35 or win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - pybind11
+    - python
+    - setuptools
+    - pip
+    - numpy >=1.16
+  run:
+    - python
+    - numpy >=1.16
```

Comment:
This should be pin_compatible. See https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy

Reply:
Done in cd25de95b